graphql/schema/types/distro.graphql

```diff
@@ -166,6 +178,20 @@ type DeleteDistroPayload {
   deletedDistroId: String!
 }
 
+
+type DistroEvent {
+  after: Map
+  before: Map
+  data: Map
```

Comment:
I think this is an anti-pattern and we should reduce our usage of the Map type. I understand why it's used here but we should try to reduce our usage/dependence on it. We lose all of the benefits of Graphql for these fields when we use it. (Nonactionable just wanted to state my thoughts)